include/iio/iio.h

```diff
@@ -1606,23 +1606,6 @@ iio_device_debug_attr_write_raw(const struct iio_device *dev,
 		 double: iio_device_debug_attr_write_double)(dev, attr, val)
 
 
-/** @brief Identify the channel or debug attribute corresponding to a filename
- * @param dev A pointer to an iio_device structure
- * @param filename A NULL-terminated string corresponding to the filename
- * @param chn A pointer to a pointer of an iio_channel structure. The pointed
- * pointer will be set to the address of the iio_channel structure if the
- * filename correspond to the attribute of a channel, or NULL otherwise.
- * @param attr A pointer to a NULL-terminated string. The pointer
- * pointer will be set to point to the name of the attribute corresponding to
- * the filename.
- * @return On success, 0 is returned, and *chn and *attr are modified.
- * @return On error, a negative errno code is returned. *chn and *attr are not
- * modified. */
-__api __check_ret int iio_device_identify_filename(const struct iio_device *dev,
-		const char *filename, struct iio_channel **chn,
-		const char **attr);
-
-
```

Comment:
I guess this function is not that simple simple to deserve an helper. But it was added so, IMHO, we now have to leave with it (or at least give time - using deprecated or so - for users of this to remove it). So, NAK from my side as I do think breaking our users should be a no go...

(BTW, I'm also not a big fan of the deprecated __attribute__)